src/Akka.Persistence.Redis.Tests/RedisFixture.cs

```diff
@@ -33,7 +33,9 @@ public RedisFixture()
             Client = config.CreateClient();
         }
 
-        protected string RedisImageName => "redis";
+        protected string ImageName = "redis";
+        protected string Tag = "6.0";
+        protected string RedisImageName => $"{ImageName}:{Tag}";
```

Comment:
This is might be your problem - just keep the image name and tag separate. Where else is this value being used?